pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -352,12 +352,7 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	envVars = append(envVars, []v1.EnvVar{
```

Comment:
Please update the commit and PR title and description 
You can refer to https://github.com/rook/rook/pull/12300 PR and we can keep this PR and close that one.

Thanks